cpp/src/IceStorm/Parser.h

```diff
@@ -10,30 +10,6 @@
 #include <list>
 #include <stdio.h>
 
-//
-// Stuff for flex and bison
-//
-
-#define YYSTYPE std::list<std::string>
-#define YY_DECL int yylex(YYSTYPE* yylvalp)
-YY_DECL;
```

Comment:
This code is very outdated. In modern Bison, these lines were replaced by the:
```
%define api.value.type {std::list<std::string>}
```
you'll find was added on line 69 of the the `Grammar.y` file.